fastplotlib/graphics/selectors/_linear_region.py

```diff
@@ -26,16 +26,22 @@ class LinearBoundsFeature(GraphicFeature):
     | "selected_indices" | ``numpy.ndarray`` or ``None`` | selected graphic data indices                                                        |
     | "selected_data"    | ``numpy.ndarray`` or ``None`` | selected graphic data                                                                |
     | "new_data"         | ``(float, float)``            | current bounds in world coordinates, NOT necessarily the same as "selected_indices". |
+    | "move_info"        | ``MoveInfo``                  | last position and event source (fill, pygfx.Mesh, or edge line)                      |
+    | "collection_index" | ``int``                       | index of collection                                                                  |
+    | "graphic"          | ``Graphic``                   | the selection graphic                                                                |
+    | "delta"            | ``numpy.ndarray``             | the delta vector of the graphic in NDC                                               |
+    | "world_object"     | ``pygfx.WorldObject``         | pygfx World Object                                                                   |
+    | "pygfx_event"      | ``pygfx.Event``               | pygfx Event                                                                          |         
```

Comment:
order the table the same for all selectors, use the online editor it makes it easy: https://tableconvert.com/restructuredtext-generator